README.md

```diff
@@ -43,30 +43,33 @@ addDecorator(Story => <MemoryRouterProvider><Story /></MemoryRouterProvider>);
 
 Simply drop-in the `next-router-mock` like this:
 
-```jsx
-jest.mock('next/router', () => require('next-router-mock'));
+file `./__mocks__/mock-next-router.ts`:
```

Comment:
Thank you for bringing this up!  I do think it makes more sense to provide "global mock" instructions!

While I do like the solution you have here, I think it might be improved by using jest's auto-mocking feature ... so maybe instead, we could recommend adding a file at `./__mocks__/next/router` or maybe `./__mocks__/next/dist/client/router`?  Then, you'd have a global mock, without requiring any other `import` or `jest.mock` calls anywhere.  

What do you think?

Reply:
The fact the user can choose two strategies (next/router OR dist/client/router) makes it little more complex to have in simple start of the readme. I just wanted to make sure what is written actually works.

Maybe its better to have example folder and have some real working examples? If i find the time i can start with one :)

Reply:
You're right, I agree ... a simple "do this" would be better, than explaining all the nuances with Next Link.
I'll take some time and reconsider the documentation, and hopefully find a clearer way to do this.  

For me, mocking `next/router` was my original use-case ... I didn't have any tests for `next/link`.  However, it does look like a lot of folks are using this mock for `next/link`, so it would be easier to support that by default.